streamlit-snowpark-dynamic-filters/lib/chart_helpers.py

```diff
@@ -0,0 +1,33 @@
+"""
+Copyright (c) 2022 Snowflake Inc.
```

Comment:
Do we want to say this code is copyrighted by Snowflake

Reply:
I guess so

Reply:
Yeah

Reply:
Sounds good!